test/support/mocks/mdm.service.mock.ts

```diff
@@ -0,0 +1,22 @@
+import { UkefSiteId } from '@ukef/helpers/ukef-id.type';
```

Comment:
I'd be tempted as mdm is using simple http endpoints to use `nock` here ala TFS. This allows us the check the request is being sent as expected, and is setting the headers etc as needed.

The reason we don't do this for Graph service is it uses an external package to make these calls. In that case, we don't care about _what_ calls it is making, rather that we are calling the external package correctly, and we mock the response based on that.

Reply:
Sounds like good practice, but mdm call is quite simple, so not sure how important it is.

Created new user story APIM-478 so we can prioritise.